package.json

```diff
@@ -181,7 +181,7 @@
     "known-css-properties": "^0.31.0",
     "mathml-tag-names": "^2.1.3",
     "meow": "^13.2.0",
-    "micromatch": "^4.0.5",
+    "micromatch": "^4.0.7",
```

Comment:
If you wanna test something, try `"4.0.6"`.
i.e. `"4.0.7"` reverted the version of picomatch

Reply:
Downgraded to 4.0.6 with 857bf8250. It can reproduce the error. 👍🏼 

So, I think that we should merge #7712 and then rebase #7713. If so, the CI error should be resolved.

Reply:
I think we will still need a resolutions for yarn and a overrides for npm.
e.g.

```
  "overrides": {
    "lint-staged": {
      "micromatch": "^4.0.7"
    }
  }
```

I cannot push on `dependabot/npm_and_yarn/lint-staged-15.2.4` without having my token being accepted.
Also I have requested to become an owner on stylelint/stylelint like @romainmenke is.